openlibrary/plugins/worksearch/subjects.py

```diff
@@ -162,6 +162,21 @@ def normalize_key(self, key):
     def process_key(self, key):
         return key
 
+class subject_history(delegate.page):
+    path = '/subjects/history'
+    encoding = 'json'
+    
+    API_ROWS_LIMIT = 100
+    @jsonapi
+    def GET(self):
+        i = web.input()
+	try:
+		limit = int(i.get("limit",self.API_ROWS_LIMIT))
+	except (ValueError, TypeError):
+		return {'error':'limit must be an integer'}
+        if limit>self.API_ROWS_LIMIT:
+            return {'error':'limit must be less than or equal to {}'.format(self.API_ROWS_LIMIT)}
+        return history().get()
```

Comment:
Does this work? I think the method is called `GET` not `get`

Reply:
OK, changed to `history().GET()`.